syft/file/contents_cataloger_test.go

```diff
@@ -0,0 +1,89 @@
+package file
+
+import (
+	"testing"
+
+	"github.com/anchore/syft/syft/source"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestContentsCataloger(t *testing.T) {
+	allFiles := []string{"test-fixtures/last/path.txt", "test-fixtures/another-path.txt", "test-fixtures/a-path.txt"}
+
+	tests := []struct {
+		name       string
+		globs      []string
+		maxSize    int64
+		files      []string
+		expected   map[source.Location]string
+		catalogErr bool
```

Comment:
`catalogErr` — This doesn't look used. If that's true, I'd remove it from the struct and remove code that uses it in the test body.